.travis.yml

```diff
@@ -1,5 +1,14 @@
 sudo: false
 language: d
+d:
+  - dmd-beta
+  - dmd
```

Comment:
Hmm, DUB will automatically detect that only ldc is available and use it to run the tests ;-)
In the Makefile `DC ?= dmd` will mean that it can be overwritten by the environment variable, so it should work out of the box.
However, feel free to add this in a further step.